package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon --exec babel-node src/index.js",
-    "test": "NODE_ENV=test && npm run createTables && mocha --require @babel/polyfill --require @babel/register src/**/**test.js",
+    "test": "NODE_ENV=test && npm run createTables && mocha --timeout 10000 --require @babel/polyfill --require @babel/register src/**/**test.js",
```

Comment:
@EugeneMunya  any particular reasons to add `timeout 10000?`

Reply:
When you are testing the database your testcase may take time to finish because it is queuing the database, to increase the time for your testcase should last before going to the next testcase you tell mocha to wait by specifying a time it should wait with timeout frag.